docs/docs/30-how-to-guides/15-working-with-freight.md

```diff
@@ -138,7 +120,21 @@ kargo update freight \
 ```
 
 This can also be accomplished via `kubectl` commands `apply`, `edit`, `patch`,
-etc.
+etc. by updating the `alias` field of the `Freight` resource.
+
+:::info
+The `alias` field is a convenient way to update the `Freight` resource's
+`kargo.akuity.io/alias` label, which causes a webhook to sync the field value
+to the label value. The precedence rules for syncing between the field and
+label values are as follows:
+
+- If the field has a non-empty value, the label will assume the field's value.
+- If the field has an empty value, the field will assume the label's value.
+  value.
```

Comment:
```suggestion
```

Reply:
I hope you don't mind that I fixed this. Figured it was not-controversial and would speed up approval. LGTM